src/docs/deployment/android.md

```diff
@@ -255,21 +258,24 @@ are correct, especially:
 
 Review the default [Gradle build file][gradlebuild] file,
 `build.gradle`, located in `<app dir>/android/app` and
-verify the values are correct, especially:
+verify the values are correct, especially the following:
 
-* `defaultConfig`:
+**`defaultConfig`**
```

Comment:
Should there be an explanation under this item? It looks like a mistake. Maybe remove it and change the previous sentence to be "especially the following values in the `defaultConfig` block"?